website/frontend/src/components/Hero.js

```diff
@@ -47,9 +48,7 @@ const Hero = () => {
             <div className="hero-content">
                 <div>
                 <p className="hero-title">Clean air for <br />all African cities </p>
-                <p className="hero-sub">We empower communities with accurate, hyperlocal and timely air quality data to
-                    drive air pollution mitigation actions.
-                </p>
+                <p className="hero-sub"> <span style={{color:"#135DFF"}}>“9 out of 10 people breathe polluted air”.</span> <br/>We empower communities with accurate, hyperlocal and timely air quality data to drive air pollution mitigation actions.</p>
                 <div className="hero-buttons">
                     <Button label="Explore data" />
                     <Button className="button-get-involved" label="Get Involved" />
```

Comment:
The buttons should be aligned to the center in the mobile design. They seem to be left-aligned.

Reply:
> The buttons should be aligned to the center in the mobile design. They seem to be left-aligned. Unless that's an issue with browser responsiveness?

@FaithDaka I have fixed that. Please re-review